<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferDialog</class>
 <widget class="QDialog" name="TransferDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Transfer</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="margin">
    <number>16</number>
   </property>
   <!-- Status -->
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Sending...</string>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>63</weight>
      </font>
     </property>
    </widget>
   </item>
   <!-- File Name -->
   <item>
    <widget class="QLabel" name="fileNameLabel">
     <property name="text">
      <string>filename.txt</string>
     </property>
    </widget>
   </item>
   <!-- Progress Bar -->
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <!-- Size Label -->
   <item>
    <widget class="QLabel" name="sizeLabel">
     <property name="text">
      <string>0 / 0 bytes</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight</set>
     </property>
    </widget>
   </item>
   <!-- Button -->
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="buttonSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
